app/views/hyrax/content_blocks/_form.html.erb

```diff
@@ -1,133 +1,145 @@
-<%# Copied from Hyrax v2.9.0 to add home_text content block form - Adding themes %>
+<%# Copied from Hyrax v5.0.0rc2 to add home_text content block form - Adding themes %>
 <%# imported from hyrax 3.0.0.pre.rc1 %>
 <%= render "shared/nav_safety_modal" %>
-<div class="panel panel-default tabs">
+<div class="card tabs">
   <ul class="nav nav-tabs" role="tablist">
-    <li class="active">
-      <a href="#announcement_text" role="tab" data-toggle="tab" class="nav-safety-confirm"><%= t(:'hyrax.content_blocks.tabs.announcement_text') %></a>
+    <li id="announcement-nav-item" class="nav-item">
```

Comment:
Reading note; my assumption is for nav-tabs that we need an id that is of the form `<WORDS>-nav-item`.

Reply:
i believe so, it is how they do it in Hyrax
https://github.com/samvera/hyrax/blob/double_combo/app/views/hyrax/content_blocks/_form.html.erb#L4